src/Microsoft.TemplateEngine.Cli/CommandParsing/NewCommandInputCli.cs

```diff
@@ -11,32 +11,30 @@ namespace Microsoft.TemplateEngine.Cli.CommandParsing
 {
     public class NewCommandInputCli : INewCommandInput
     {
-        private ParseResult _parseResult;
```

Comment:
Is there anything that you think we should review here or is it just refactoring? Too many changes to review the whole file 🙂

Reply:
there was no changes to that file in my previous commit, just members sorting. 
Overall the only change is adding this property:
```
public string AuthorFilter => _parseResult.GetArgumentValueAtPath(new[] { _commandName, "author" });
```